tables/teams.tex

```diff
@@ -5,50 +5,50 @@
-{3.2k}&{Infrastructure and Support}&{Wide Area Network Technical Manager}&{Responsible for providing coordination amongst and managing relationships with the four independent WAN operators. Acts as the interface for services provided to the US DF in the context of the WAN. Responsible for managing the risk associated with each WAN operator, including developing mitigation strategies and proposed project responses to credible risks. Leads the Joint Wide Area Network Working Group. Well connected to DOE ESNet.}&{Fermilab}&{0.37} \\ \hline
-{3.2l}&{Infrastructure and Support}&{Wide Area Network Architect}&{Familiar with WAN implementation technologies generally available in the networks supporting the Rubin Observatory. Familiar with technology roadmap of the ESNet WAN provider. Synthesizes and evolves network techniques and provisioning supporting the Rubin Observatory mission, as network technology evolves. Drawn from staff of WAN groups but explicitly supported by and work in the context of the Rubin Observatory.}&{Fermilab}&{0.75} \\ \hline
+{3.2k}&{Infrastructure and Support}&{Wide Area Network Technical Manager}&{Responsible for providing coordination amongst and managing relationships with the four independent WAN operators. Acts as the interface for services provided to the US DF in the context of the WAN. Responsible for managing the risk associated with each WAN operator, including developing mitigation strategies and proposed project responses to credible risks. Leads the Joint Wide Area Network Working Group. Well connected to DOE ESNet.}&{Fermilab}&{0.38} \\ \hline
+{3.2l}&{Infrastructure and Support}&{Wide Area Network Architect}&{Familiar with WAN implementation technologies generally available in the networks supporting the Rubin Observatory. Familiar with technology roadmap of the ESNet WAN provider. Synthesizes and evolves network techniques and provisioning supporting the Rubin Observatory mission, as network technology evolves. Drawn from staff of WAN groups but explicitly supported by and work in the context of the Rubin Observatory.}&{Fermilab}&{0.38} \\ \hline
+{3.2r}&{Infrastructure and Support}&{Data Wrangler - IN2P3}&{The data wrangler ensures that data (science raw data, calibration data, data products, etc.) is replicated at IN2P3 and data products resulting from the local processing performed at IN2P3 are replicated to the USDF They also ensure that the data archived at IN2P3 and needed for the annual processing are recalled from tape on time for the image processing tasks to be performed. They ensure that the tools and systems used for replicating data at IN2P3 are operational. This role needs tight coordination with the team at the USDF that is responsible for data distribution.}&{IN2P3}&{0.35} \\ \hline
```

Comment:
These hyphens should all be at least en-dashes.

Reply:
this is generated from a google sheet .. i guess i can make it do a(nother) substitution